plugins/make_rpm

```diff
@@ -222,7 +222,11 @@ EOF
 			if [ -f requirements.sh ]; then
 				./requirements.sh
 			fi
-			mkdir -p build; cd build; cmake ..; make)
+			if [ -f build_rhel.sh ]; then
```

Comment:
Just because it is an rpm we are building it does not mean it is RHEL, Debian RPM's can be built as well